_posts/2019-02-20-lecture-notes-template.md

```diff
@@ -0,0 +1,361 @@
+
+
+## Discussion of Where the A, G, and C Matrix Come From 
+
+
+Up to this point, we have discussed inference in the Kalmann filter model; given the 
+model up-front, tell me something about the data. This leaves open where the matrices A, G, and
+C come from, however. This is a similar situation we were in for HMMs: to find the necessary 
+matrices, we must do learning. Using approaches like EM we can interleave learning and inference
+to come across the parameters of interest.
+
+Furthering this comparison to HMMs, the Rauch-Tung-Strievel algorithm allows us to perform "exact off-line inference in an LDS", and is essentially a "Guassian analog of the forwards-backwards" algorithm.
+
+## Learning SSMs
+In order to learn the necessary parameters for the Kalmann filter, 
+we calculate the complete data likelihood:
+$$l_t(\theta, D) = \sum_{n}p(x_n, y_n) = $$
+$$(\sum_n log(p(x_1))) + (\sum_n\sum_t log(p(x_{n,t} | x_{n, t-1})) + (\sum_n\sum_t log(p(y_{n,t} | x_{n, t}))) =$$
+$$f_1(X; \Sigma_0) + f_2(\{X_tX_{t-1}^{T}, X_tX_t^{T}, X_t: \forall t\}, A, Q, G) + 
+f_3(\{X_tX_t^{T}, X_t: \forall t\}, C, G) $$
+
+
+This is very similar to what we saw in factor analysis, except there, we 
+computed this for each individual time-step, whereas here we do it 
+for all time-steps. From here, we proceed as usual in EM:
+in the E-step, we estimate $$X_tX_{t-1}^{T}$$, $$X_tX_t^{T}$$, and $$X_t$$
+by taking their expectation in respect to the observation, and in the 
+M-step we use MLE in the typical fashion. 
+
+## Non-Linear Systems
+
+The approaches discussed thus far are designed to handle linear systems.
+In order to handle a non-linear system, an approximation must be made. 
+For example, for non-linear differentiable systems, Taylor expansion
+allows us to use linear terms to approximate non-linear curves.
+
+With that, we close the modeling section of the course and move on to the 
+next subject.
+
+### Approximate Inference and Topic Models: Mean Field and Loopy Belief Prop
+
+Thus far in the course, we have covered the elimination algorithm, message passing, and
+algorithms that are powered based on those principles. These techniques allowed for 
+exact inference, and while they have been shown useful, they cannot cope with many of 
+the settings and problems we wish to explore. This motivates uses of approximate
+techniques, which we will begin discussing in this lecture.
+
+
+
+
+## Some Discussion of a Task, and How to Tackle It with Appropriate Modeling
```

Comment:
"Appropriately Modeling Different Tasks"

Reply:
Done. Saw this comment in the email I received and addressed it then, but did not see it here earlier.